enterprise/api/v1/views/enterprise_group.py

```diff
@@ -237,6 +249,9 @@ def assign_learners(self, request, group_uuid):
                 'new_learners': total_new_users_processed,
                 'existing_learners': total_existing_users_processed,
             }
+            for membership in memberships:
+                    if budget_expiration is not None and catalog_uuid is not None:
+                        send_group_membership_invitation_notification.delay(group.enterprise_customer, membership, budget_expiration, catalog_uuid)
```

Comment:
why are we sending the invitation notification twice in assign learners?

Reply:
good catch. I think we should keep this loop here since it applies to all memberships and remove the `send_group_membership_invitation_notification` function up above where we are looping over `for ent_customer_user in ent_customer_users` since that one is only sending to existing learners.